src/coreclr/vm/interpexec.cpp

```diff
@@ -1129,6 +1298,7 @@ void InterpExecMethod(InterpreterFrame *pInterpreterFrame, InterpMethodContextFr
         goto MAIN_LOOP;
     }
 
+    frame_data_allocator_destroy(&pThreadContext->frameDataAllocator);
```

Comment:
This data structure is owned by the current thread, it is meant to be freed when the thread detaches. We don't need to free it here for every transition into the interpreter (having to malloc/free every time). A FIXME so we can address this later for perf. Currently we don't free this interp thread specific data (like the interp stack).

Reply:
Added, thanks.